packages/frontend/src/renderer/containers/widgets/joinCommunity/joinCommunity.basic.test.tsx

```diff
@@ -126,65 +126,40 @@ describe('join community', () => {
     expect(createUsernameTitle).toBeVisible()
   })
 
-  it('user rejoins to remembered community with certificate', async () => {
+  it('user rejoins to remembered community with user certificate', async () => {
     const factoryStore = (await prepareStore()).store
     const factory = await getFactory(factoryStore)
 
     const community = await factory.create<
       ReturnType<typeof communities.actions.addNewCommunity>['payload']
     >('Community')
+    await factory.create<
```

Comment:
you texted before that i should use factory girl-for that too, not an object, so i used it and next i'm dispatching action that sets user certificate as ' '

Reply:
And I mean it, of course you should use factory-girl. The thing is it can be used in various ways. Now I see you indeed override the certificate by posting it under the same communityId so it's fine